marklogic-data-hub-central/gradle.properties

```diff
@@ -5,8 +5,8 @@ springBootJarName=marklogic-data-hub-central
 # For 5.4.0, Spring Boot 2.3.x is being used to ensure that it uses Jackson 2.11, which is the same version that 
 # the ML Java Client wants to use. Spring Security 5.3.8 is then used to ensure it and Spring Boot are using the same
 # underlying Spring libraries, which is version 5.2.12. 
-springBootVersion=2.3.8.RELEASE
-springSecurityVersion=5.3.8.RELEASE
+springBootVersion=2.3.11.RELEASE
```

Comment:
The comment above needs to be updated based on these new version numbers - probably better to reword it like this so it doesn't have version numbers:

"Spring Security and Spring Boot are then configured with version numbers to ensure they use the same underlying Spring library version."